annif/transformer/inputlimiter.py

```diff
@@ -0,0 +1,22 @@
+# TODO Add docstring
+from annif.exception import ConfigurationException
+from . import transformer
+
+
+class InputLimiter(transformer.IdentityTransformer):
+
+    name = 'limit_input'
+
+    def __init__(self, project, input_limit):
+        self.project = project
+        self.input_limit = int(input_limit)
```

Comment:
Not that Annif generally does much validation of configuration values, but...this could be improved by passing the original `input_limit` string to `_validate_value` which would convert it to int (throwing a ConfigurationException if that fails, for example if the value wasn't a number) and then also testing the value.

Also what happens if the InputLimiter transform is used without a parameter, or with more than one? Maybe that's covered by the "Invalid arguments to..." exception below?

Reply:
My idea was that the common argument validation by ConfigurationException with "Invalid arguments to..."  would handle both type validation of arguments (ValueErrors) and also missing (or too many) arguments (TypeErrors)  to transforms. 

For example for missing character number to limit transform:
`Error: Misconfigured project 'yso-yake-fi': Invalid arguments to limit transform: [], {})`

Or for wrong type for character number:
`Error: Misconfigured project 'yso-yake-fi': Invalid arguments to limit transform: ['not-a-number'], {})`

 Each transformer could then also do more specific validation for e.g. value ranges as in here.